src/authutils/oauth2/client/client.py

```diff
@@ -1,4 +1,4 @@
-from authlib.client import OAuthClient
+from authlib.oauth2.client import OAuth2Client
```

Comment:
thanks for the contribution 🙂 how did you run into this issue? the current code seems to be working fine in our services

Reply:
chicagopcdc/amanuensis imports authutils and authlib.  I was attempting to upgrade to newer releases and encountered the error because 'OAuthClient' doesn't exist in more recent releases of authlib. I checked 0.15.3 and 0.15.5 specifically.